### PR TITLE
Bugfix/816 jamulus 2723 self-labelled checkboxes

### LINKED ISSUE #816: Checkbox controls should be "self-labelling"
Following up on https://github.com/jamulussoftware/jamulus/issues/2723#issuecomment-1239293096 so it doesn't get lost.

That issue has initial screenshots.  Once they're agreed, I'll post the final ones here along with any further changes needed in the "Software Manual" and "Running a Server" pages.

### PR DESCRIPTION
**Pull request by pljones**
**Short description of changes**
Update Jamulus manual for "self-labelled checkboxes" change, jamulussoftware/jamulus#2723.

**Context: Fixes an issue? Related issues**
Fixes: 816
See jamulussoftware/jamulus#2723 and jamulussoftware/jamulus#2996 for more details.

**Status of this Pull Request**
Work in progress, raised to create a tarball so I can get the latest state for testing.

**What is missing until this pull request can be merged?**
- [x] ~I need to add the server screenshots.~ (These appear to have been removed from the wiki.)
- [x] I need to look through the wiki text for references.
- [x] I need to check whether there are any other screenshots needing to be updated.
- [x] Then I'll move the PR out of draft so it'll be ready for review.

**Does this need translation?**
YES

## Checklist
- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by pljones** [comment]
I'm pushing my changes to http://jamuluswebsite.drealm.info/wiki/Software-Manual for easier review.

File: wiki/en/Software-Manual.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ If the Auto setting is enabled, the jitter buffers
 of your local Client and the remote server are set automatically based on measurements of the network and sound card
 timing jitter. If the Auto check is enabled, the jitter buffer size faders are disabled (they cannot be moved with the mouse).
 
-### Enable small network buffers
+### Small Network Buffers
```
Comment (ann0see) [inline wiki/en/Software-Manual.md:227]: This could break some links, but should also show up if we run the linkchecker.

**Review by ann0see — APPROVED** [review]
I believe it's ok.

File: wiki/en/Software-Manual.md (hunk 1 above)
Comment (pljones) [reply wiki/en/Software-Manual.md:227]: `$ grep -r 'Enable small network buffers' wiki/en* assets/ *html *xml` comes back with nothing, at least.

**Comment by ann0see** [comment]
Merging this now.

**Pull request merged**